plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/ast/Stmt.kt

```diff
@@ -96,34 +96,6 @@ sealed interface Stmt : IntoSilver<viper.silver.ast.Stmt> {
         )
     }
 
-    data class Assert(
-        val exp: Exp,
-        val position: Position = Position.NoPosition,
-        val info: Info = Info.NoInfo,
-        val trafos: Trafos = Trafos.NoTrafos,
-    ) : Stmt {
-        override fun toSilver(): viper.silver.ast.Assert = viper.silver.ast.Assert(
-            exp.toSilver(),
-            position.toSilver(),
-            info.toSilver(),
-            trafos.toSilver()
-        )
-    }
```

Comment:
Are we sure that `Assert` is also not supported?

Reply:
Ah, looks like it is.  I guess we might want to use it eventually, kept it in.